ocaml/testsuite/tests/typing-implicit-source-positions/let_operators.ml

```diff
@@ -0,0 +1,52 @@
+(* TEST
+   * expect
+*)
+
+let ( let+ ) ~(src_pos : [%src_pos]) a f = f (src_pos, a);;
+[%%expect{|
+val ( let+ ) : src_pos:[%src_pos] -> 'a -> (lexing_position * 'a -> 'b) -> 'b =
+  <fun>
+|}]
+
+(* CR src_pos: Add support for implicit position parameters and (also maybe optional
```

Comment:
```suggestion
(* Would be nice to add support for implicit position parameters and (also maybe optional
```

I think we should leave off the `CR`. This isn't something we'll come back to as part of this project, and we'd be content even upstreaming the code as-is (if upstream is content).

Reply:
I have updated the comment! Thanks!